internal/cli/test.go

```diff
@@ -157,46 +123,63 @@ func testLoginCmd(cli *cli) *cobra.Command {
 				return nil
 			}
 
+			if inputs.Audience != "" {
```

Comment:
Regular Web Apps could be authorized to request access tokens for a specific API so if we do pass an audience we wanna make sure the application is authorized.